pkg/controllers/webhookauthenticator/webhookauthenticator_controller.go

```diff
@@ -83,7 +87,7 @@ func NewWebhookAuthenticatorController(
 	).ResyncEvery(wait.Jitter(time.Minute, 1.0)).
 		WithSync(c.sync).
 		WithSyncDegradedOnError(operatorClient).
-		ToController("WebhookAuthenticatorController", recorder.WithComponentSuffix("webhook-authenticator-controller"))
+		ToController(instanceName, recorder.WithComponentSuffix("webhook-authenticator-controller"))
```

Comment:
I think that we cannot rename this controller without removing existing condition added by the `WithSyncDegradedOnError`.

Reply:
Actually we aren't renaming the controller, it's got the same name here:  https://github.com/openshift/cluster-authentication-operator/pull/698/files#diff-0d623dfd885adb20f991bda4c2453aebd732ca6dbb4d1d4be6e79805c3b48de6R720

Reply:
ok, then please add a comment where you set the name to not change it without clearing the condition.

Reply:
for example: https://github.com/openshift/library-go/blob/ac65cd59c78f04a82617f7aecd6a0f1d16f985cb/pkg/operator/apiserver/controller/auditpolicy/auditpolicy_controller.go#L62

Reply:
Good idea; done